src/publicStore.js

```diff
@@ -12,6 +12,16 @@ class ProfileStore extends KeyValueStore {
     this._linkProfile()
     return super.set(key, value)
   }
+
+  async setMultiple(keys, values) {
```

Comment:
I think this method should live in the `KeyValueStore` class with a minimal wrapper here.